[2022-05-30 20:21:25,385] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-11-02T06:00:00+00:00 [queued]>
[2022-05-30 20:21:25,463] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-11-02T06:00:00+00:00 [queued]>
[2022-05-30 20:21:25,465] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 20:21:25,468] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-30 20:21:25,471] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 20:21:25,679] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): ingest> on 2021-11-02 06:00:00+00:00
[2022-05-30 20:21:25,856] {standard_task_runner.py:52} INFO - Started process 1286 to run task
[2022-05-30 20:21:25,890] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-11-02T06:00:00+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmp_uj1t7j1', '--error-file', '/tmp/tmplztloed5']
[2022-05-30 20:21:25,953] {standard_task_runner.py:80} INFO - Job 135: Subtask ingest
[2022-05-30 20:21:26,634] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-11-02T06:00:00+00:00 [running]> on host e083cbf9476b
[2022-05-30 20:21:27,456] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-30 20:21:29,197] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T06:00:00+00:00
[2022-05-30 20:21:29,315] {logging_mixin.py:115} INFO - yellow_taxi_2021_11 /opt/***/output_2021-11.parquet 2021-11-02T06:00:00+00:00
[2022-05-30 20:21:29,339] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ingest_script.py", line 12, in ingest_callable
    engine = create_engine(f'postgresql://{user}:{password}@{host}:{port}/{db}')
  File "<string>", line 2, in create_engine
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/deprecations.py", line 298, in warned
    return fn(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/create.py", line 518, in create_engine
    u = _url.make_url(url)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/url.py", line 711, in make_url
    return _parse_rfc1738_args(name_or_url)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/url.py", line 772, in _parse_rfc1738_args
    components["port"] = int(components["port"])
ValueError: invalid literal for int() with base 10: 'None'
[2022-05-30 20:21:29,680] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20211102T060000, start_date=20220530T202125, end_date=20220530T202129
[2022-05-30 20:21:30,552] {standard_task_runner.py:97} ERROR - Failed to execute job 135 for task ingest (invalid literal for int() with base 10: 'None'; 1286)
[2022-05-30 20:21:31,860] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-05-30 20:21:32,285] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-30 22:17:50,129] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-11-02T06:00:00+00:00 [queued]>
[2022-05-30 22:17:50,185] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-11-02T06:00:00+00:00 [queued]>
[2022-05-30 22:17:50,187] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:17:50,190] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-30 22:17:50,191] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:17:50,246] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): ingest> on 2021-11-02 06:00:00+00:00
[2022-05-30 22:17:50,352] {standard_task_runner.py:52} INFO - Started process 1443 to run task
[2022-05-30 22:17:50,437] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-11-02T06:00:00+00:00', '--job-id', '225', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpxtly2emm', '--error-file', '/tmp/tmpp_mvdnxm']
[2022-05-30 22:17:50,451] {standard_task_runner.py:80} INFO - Job 225: Subtask ingest
[2022-05-30 22:17:50,850] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-11-02T06:00:00+00:00 [running]> on host 8ac972de2f4f
[2022-05-30 22:17:51,013] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-30 22:17:51,224] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T06:00:00+00:00
[2022-05-30 22:17:51,254] {logging_mixin.py:115} INFO - yellow_taxi_2021_11 /opt/***/output_2021-11.parquet 2021-11-02T06:00:00+00:00
[2022-05-30 22:17:51,550] {logging_mixin.py:115} INFO - connection established successfully, inserting data...
[2022-05-30 22:18:36,805] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGKILL
[2022-05-30 22:18:37,083] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20211102T060000, start_date=20220530T221750, end_date=20220530T221837
[2022-05-30 22:18:37,318] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
